docs/d3-component-framework.rst

```diff
@@ -1,277 +0,0 @@
-======================
```

Comment:
Is this no longer relevant? Don't we still use d3?

Reply:
We do but we no longer use this architecture so this information was incorrect.